src/content/components/FeatureList/FeatureList.js

```diff
@@ -63,6 +63,7 @@ export class FeatureList extends React.PureComponent {
         {this.renderTableHead("Meta Bug", `Prioritized for Firefox ${release}`)}
         {this.renderTableBody(prioritized)}
         {this.renderTableHead("", "Other features")}
+        <tr><td className={styles.idColumn} /><td className={styles.displayNameColumn}> <Link to={"/no-feature"}>[No feature]</Link></td></tr>
```

Comment:
Nit: when this is rendered the `<tr>` ends up being a direct child of a `<table>` and we get an ugly console warning:

```
Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody> to your code to match the DOM tree generated by the browser.
```

Wrapping this line in its own `<tbody>` seems to fix things though.